compiler/packages/babel-plugin-react-compiler/src/Validation/ValidateHooksUsage.ts

```diff
@@ -452,7 +452,7 @@ function visitFunctionExpression(errors: CompilerError, fn: HIRFunction): void {
                 reason:
                   'Hooks must be called at the top level in the body of a function component or custom hook, and may not be called within function expressions. See the Rules of Hooks (https://react.dev/warnings/invalid-hook-call-warning)',
                 loc: callee.loc,
-                description: `Cannot call ${hookKind} within a function component`,
+                description: `Cannot call ${hookKind === 'Custom' ? 'custom hook' : hookKind} within a function component`,
```

Comment:
Nit: just drop the `custom`? I don't think it's adding information here.

```suggestion
                description: `Cannot call ${hookKind === 'Custom' ? 'hook' : hookKind} within a function component`,
```

Also what's the difference bewteen reason and description? If they're concatenated it's a bit duplicative.

Reply:
@kassens we use reason in the internal scripts for categorization, so it can be grouped while description is more for specific debugging info